subprojects/docs/src/docs/release/notes.md

```diff
@@ -87,7 +87,144 @@ TODO - Java lambdas are supported, and unsupported captured values are reported.
 TODO - File collections queried at configuration time are treated as configuration inputs.
 TODO - File system repositories are fully supported including dynamic versions in Maven, Maven local, and Ivy repositories
 
-#### Build Init plugin incubating option changes
+### Kotlin DSL improvements
+
+Gradle's [Kotlin DSL](userguide/kotlin_dsl.html) provides an alternative syntax to the Groovy DSL with an enhanced editing experience in supported IDEs — superior content assistance, refactoring, documentation and more.
+
+#### Version catalogs for plugins in the `plugins {}` block
+
+Version catalog accessors for plugin aliases in the `plugins {}` block aren't shown as errors in IntelliJ IDEA and Android Studio Kotlin script editor anymore.
+
+```kotlin
+plugins {
+    alias(libs.plugins.jmh)
+}
+```
+
+If you were using a workaround for this before, see the [corresponding section](userguide/upgrading_version_8.html#kotlin_dsl_plugins_catalogs_workaround) in the upgrading guide.
+
+#### Easier access to extensions on the `Gradle` object 
+
+The `Gradle` type now declares that it is `ExtensionAware`.
```

Comment:
💭 
Do a lot of people use this? It might be nice to list why it is useful in init scripts?

Reply:
I would say not a lot of people use this. This is mostly for init scripts as written two lines below.

Do you still think this needs more details?

Reply:
Considering that this is a good level of detail but I would move it to later in the section.